dist/autocomplete.angular.js

```diff
@@ -77,8 +77,8 @@
 	_.mixin = angular.extend;
 	_.Event = angular.element.Event;
 
-	var EventBus = __webpack_require__(5);
```

Comment:
Can you remove the dist/ from the git diff? It will be generated on release. (It eases reviews and merges overall)